spec/lib/os_places_api/client_spec.rb

```diff
@@ -270,4 +233,23 @@
       end
     end
   end
+
+  describe "#update_postcode" do
+    context "the postcode is outdated in the database" do
+      before do
```

Comment:
Nitpick: as we're only using this in one test, we can move this code into the test itself, and get rid of the `before` block.